pool/app/admin/entity.ml

```diff
@@ -1,15 +1,29 @@
 module Id = Pool_common.Id
 
-type t = Sihl_user.t [@@deriving eq, show]
+type t =
+  { user : Sihl_user.t
+  ; import_pending : Pool_user.ImportPending.t
```

Comment:
Do we add the "last_login" etc. in an additional issue?

Reply:
The functionality is already implemented. It is just not added to the admin model, as I wasn't sure about the use cases. When is this information actually required?